rhizome/host/bin/verify-sshable.rb

```diff
@@ -0,0 +1,41 @@
+#!/bin/env ruby
+# frozen_string_literal: true
```

Comment:
I believed we decided to not put `.rb` extension to our bin scripts. cc @fdr

Reply:
That's right, legacy "it's spring 2023 and nothing exists" stuff.